actions/bind

```diff
@@ -42,6 +43,22 @@ def parse_arguments():
     return parser.parse_args()
 
 
+def subcommand_setup(_):
+    """Setup BIND configuration."""
+    actions.superuser_run('bind', ['setup'])
```

Comment:
Looks pretty good except for this one line.

This line is the one that actually runs "bind setup", so it would make an infinite loop here...

I would keep the default_config() function in the plinth bind module, and have it only run "bind setup" (i.e. move this line to that function).

So the execution order will look like this:
- After packages are installed, the bind module's setup() will be called.
- setup() uses helper to call default_config().
- default_config() runs "bind setup" as superuser action.
- bind action script calls subcommand_setup(), which actually modified the config file.